docs/platform/howto/byoc/create-custom-cloud.rst

```diff
@@ -1,49 +1,460 @@
+
+.. note::
+    
+    Enabling :doc:`the BYOC feature </docs/platform/concepts/byoc>` or creating custom clouds in your Aiven environment does not affect the configuration of your existing organizations, projects, or services. This only makes the new BYOC capabilities available in your environment.
 
-This article provides you with instructions on how to :ref:`add a custom cloud <create-cloud>` to your Aiven organization.
+This article provides you with instructions on how to :ref:`create a custom cloud <create-cloud>` for your Aiven organization.
 
 About creating a custom cloud
 -----------------------------
 
-If you have the administrator's role in your Aiven organization, and you enable BYOC, you can create a custom cloud on the Aiven platform. For this purpose, you'll need to configure your custom cloud setup in `Aiven Console <https://console.aiven.io/>`_ and prepare your own Amazon Web Services (AWS) account so that Aiven can access it.
+.. important::
+
+   Before getting down to creating a custom cloud, make sure you understand all the :ref:`limitations <byoc-limitations>` and meet all the :ref:`prerequisites <byoc-prerequisites>`.
+
+The process of creating a custom cloud in Aiven differs depending on what cloud provider you want to integrate with:
+
+* If you want to use the AWS cloud provider, you create your custom cloud yourself in `Aiven Console <https://console.aiven.io/>`_.
 
-In `Aiven Console <https://console.aiven.io/>`_, you'll use the **Create custom cloud** workflow to generate a Terraform infrastructure-as-code (IaC) template. Next, you'll deploy this template in your AWS account to acquire Role ARN (Amazon Resource Name). You'll supply your Role ARN into the **Create custom cloud** workflow, which will give Aiven the permissions to securely access your AWS account, create resources, and manage them onward. Finally, you'll assign projects and add customer contacts for your custom cloud.
+.. topic:: BYOC self-service in Aiven Console
+
+   You configure your custom cloud setup in `Aiven Console <https://console.aiven.io/>`_ and prepare your own AWS account so that Aiven can access it. In `Aiven Console <https://console.aiven.io/>`_, you follow the **Create custom cloud** workflow to generate a Terraform infrastructure-as-code (IaC) template. Next, you deploy this template in your AWS account to acquire Role ARN (Amazon Resource Name). You supply your Role ARN into the **Create custom cloud** wizard, which gives Aiven the permissions to securely access your AWS account, create resources, and manage them onward. Finally, you select projects that can use your new custom clouds for creating services, and you add customer contacts for your custom cloud.
```

Comment:
```suggestion
   You configure your custom cloud setup in `Aiven Console <https://console.aiven.io/>`_ and prepare your own AWS account so that Aiven can access it. In `Aiven Console <https://console.aiven.io/>`_, you follow the **Create custom cloud** workflow to generate a Terraform infrastructure-as-code (IaC) template. Next, you deploy this template in your AWS account to acquire IAM Role ARN (Amazon Resource Name). You supply your IAM Role ARN into the **Create custom cloud** wizard, which gives Aiven the permissions to securely access your AWS account, create resources, and manage them onward. Finally, you select projects that can use your new custom clouds for creating services, and you add customer contacts for your custom cloud.
```